kids-first/kf-portal-ui
# fix: #SKFP-320 fix profile is reset on logout

# [BUG] [Members] Profile Status changes upon login

- closes [SKFP-320](https://d3b.atlassian.net/browse/SKFP-320)

## Description

When logged in and you change the profile status from "Private" to "Public" , the status will revert back to "Private" if you log out and log back in.

## Acceptance Criterias

The desired behaviour is that the Profile status should not be changed if a user logs out. 

## Validation



## QA

Log in, set profile public if it was private or private if it was public then log out then log in again, profile should be in latest state (public if you put it as public, private if you put it as private).
